.travis.yml

```diff
@@ -14,17 +14,17 @@ cache:
 
 env:
   matrix:
-    - SYMFONY_VERSION=2.8.* 
+    - SYMFONY_VERSION=2.8.*
   global:
     - SYMFONY_DEPRECATIONS_HELPER=weak
 
 matrix:
   include:
-    - php: 5.6
```

Comment:
no, 5.6 is still tested (see the PHP version list). However, when testing all different supported Symfony versions, we should use the latest version around (as it's the quickest).